src/accordion/pup-e2e.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+/*eslint-env node*/
+
+describe('Google', () => {
+  beforeAll(async () => {
+    await page.goto('https://google.com');
+  });
+
+  it('should display "google" text on page', async () => {
```

Comment:
Should we add a simple accordion test instead?

Reply:
yeah, that makes sense